packages/frontend/src/MobileRoutes.tsx

```diff
@@ -41,7 +41,7 @@ import MobileSpaces from './pages/MobileSpaces';
 import Projects from './pages/Projects';
 import ShareRedirect from './pages/ShareRedirect';
 import { TrackPage } from './providers/TrackingProvider';
-import { ReactComponent as Logo } from './svgs/logo-icon.svg';
+import Logo from './svgs/logo-icon.svg?react';
```

Comment:
Different syntax for vite or for something else?

Reply:
yeah, it's an svgr plugin upgrade thing

Reply:
if you omit `?react` it will import as asset (uri string)